masterlist.yaml

```
@@ -22755,6 +22755,19 @@ plugins:
       - 'Unique Landscapes.esp'
       - 'xulImperialIsle.esp'
 
+  - name: 'ShippingOnNiben.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/41134/'
+        name: 'Shipping On Niben'
+    after:
+      - 'Better Cities FULL.esp'
+      - 'Better Cities BRAVIL.esp'
+      - 'Better Cities LEYAWIIN.esp'
+      - 'Better Imperial City.esp'
+      - 'Better Cities IC Waterfront.esp'
+      - 'Better Cities - IC Imperial Isle.esp'
+      - 'Better Cities - Open Better Cities.esp'
```

```suggestion
      - 'Better Cities FULL.esp'
      - 'Better Cities - IC Imperial Isle.esp'
      - 'Better Cities LEYAWIIN.esp'
```

These are the only three plugins in this list that actually conflict with the mod (`Better Cities - Open Better Cities.esp` seems to have been dropped in favor of an in-game configuration system, so no need to add a rule for it anymore). I've also sorted them alphabetically (i.e., ignoring characters like hyphens unless they break ties).

---

Shipping On Niben was designed for an older version of Better Cities which still did include `Better Cities - Open Better Cities.esp` so I am loathe to remove that from afters. The rest, on the other hand, no problem.